src/ert/gui/simulation/run_dialog.py

```diff
@@ -401,3 +401,10 @@ def toggle_detailed_progress(self):
     def _on_finished(self):
         for file_dialog in self.findChildren(FileDialog):
             file_dialog.close()
+
+    def keyPressEvent(self, q_key_event):
+        if q_key_event.key() == Qt.Key_Escape:
+            # self.kill_button.click()
+            self.close()
+        else:
```

Comment:
Unsure what to do with the other key presses? Is there any other uses for them? Pressing enter will select the "blue" button at least.

Reply:
Usually there is a selected button/element that would be triggered if ENTER-key was pressed. I guess that is reasonable as-is today. What seems to be missing is the possibility to tab between the elements. There is usually a predetermined sequence that allows users to tab between a selected set of elements.
Regarding the enter key though, id argue that passing on the signal (like you do) makes most sense.